tests/testthat/test-loading_workbook.R

```diff
@@ -923,6 +923,14 @@ test_that("Loading multiple pivot tables: loadPivotTables.xlsx works",{
   # Check number of 'pivotCacheDefinition'
   expect_equal(length(wb$pivotDefinitions),
                2)
-  
+})
+
+test_that("Load and saving a file with Threaded Comments works", {
+  ## loadThreadComment.xlsx is a simple xlsx file that uses Threaded Comment.
+  fl <- system.file("extdata", "loadThreadComment.xlsx", package = "openxlsx")
+  wb <- loadWorkbook(fl)
+  # Check that wb can be saved without error
+  expect_error(saveWorkbook(wb, file = tempfile()),
```

Comment:
doesn't this expect an error? Maybe `expect_silent` or `expect_output_file`?

Reply:
In fact there is no function `expect_no_error()` in `testthat` and and one of the way to test it is to use `expect_error()` with argument `regexp = NA`.

For `expect_error()` documentation:

> [...] If NA, asserts that there should be no errors.[...]

But you are right `expect_silent()` is probably a more readable option here.

Reply:
Did not know that!